src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+module.exports = function(options, undef) {
+  'use strict';
+
+  const TAU = 2 * Math.PI,
```

Comment:
as a new PR, I'm simply filing everything we discussed "as text" where they apply to the code, so think of all of these as administrative comments (i.e.: "a change is needed here") that we can use for tracking purposes, not "I told you to change these, why haven't you changed them yet". If any of these lines still show up after the final rewrite, then we know we've forgotten something. It's a nice feature of github PRs.